Zend/tests/bug70644_2.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
```

Comment:
this should definitely go to json or there need to be at least skipif json ext is not there